scripts/build-crc.py

```diff
@@ -0,0 +1,100 @@
+#!/usr/bin/env python
+
```

Comment:
copyright/license blurb

Reply:
We should not validate checksums by default. This should be opt-in (maybe via `crc_check_chance`) and the tests in question should simply not opt-in (or opt-out).